src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,99 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+
+// typography
+$pf-chart-global--FontSize--xs: $pf-global--FontSize--xs; // Axis Labels
+$pf-chart-global--FontSize--sm: $pf-global--FontSize--sm; // For Legend
+$pf-chart-global--FontSize--2xl: $pf-global--FontSize--2xl; // For Large Numbers
+
+// label
+$pf-chart-global-label--FontSize: $pf-chart-global--FontSize;
+$pf-chart-global-label--Padding: 10px;
+
+// Layout Props
+$pf-chart-global-layout--Padding: 50px;
+$pf-chart-global-layout--Height: 300px;
+$pf-chart-global-layout--Width: 450px;
+
+// Individual Charts
+
+// Area Chart
+$pf-chart-area--Opacity: .4;
+$pf-chart-area-stroke--Width: 2px;
+
+// Axis Chart
+$pf-chart-axis-axis-stroke--Width: 1px;
+$pf-chart-axis-axis-label--Padding: 100px;
+$pf-chart-axis--ticks--Width: 1px;  // this should be a var for size so not sure if width is appropriate
+
+// Bar Chart
+$pf-chart-bar--Width: 10px;
+
+// Box Plot Chart
+$pf-chart-boxplot-max--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-max-stroke--Width: 1px;
+$pf-chart-boxplot-median--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-median-stroke--Width: 1px;
+$pf-chart-boxplot-min--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-min-stroke--Width: 1px;
+$pf-chart-boxplot-q1--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-q3--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-box--Width: 20px;
+
+// Bullet Chart
+$pf-chart-bullet--comparative-measure-line-warning--BorderColor: $pf-color-orange-300;
+$pf-chart-bullet--comparative-measure-line-warning--BorderWidth: 1px;
+$pf-chart-bullet--comparative-measure-line-error--BorderColor: $pf-color-red-100;
+$pf-chart-bullet--comparative-measure-line-error--BorderWidth: 1px;
+$pf-chart-bullet--zero-line--BorderColor: $pf-color-black-700;
+$pf-chart-bullet--zero-line--BorderWidth: 1px;
+$pf-chart-bullet--negative-primary-measure--BorderColor: $pf-color-red-100;
+$pf-chart-bullet--negative-primary-measure--BorderWidth: 2px;
+
+// Candlestick
+$pf-chart-candelstick-data-stroke--Width: 1px;
+
+// Chart
+$pf-chart-chart--BorderWidth: 1px;
+
+// Donut Chart
+$pf-chart-donut-color-threshold-first: $pf-color-black-200;
+$pf-chart-donut-color-threshold-second: $pf-color-black-300;
+$pf-chart-donut-color-threshold-third: $pf-color-black-400;
+$pf-chart-donut-color-threshold-success: $pf-chart-color-blue-300;
+$pf-chart-donut-color-threshold-warning: $pf-chart-color-gold-400;
+$pf-chart-donut-color-threshold-danger: $pf-color-red-100;
+
+// Error Bar
+$pf-chart-errorbar--BorderWidth: $pf-global--spacer--sm;
+$pf-chart-errorbar-data-stroke--Width: 2px;
+
+// Legend
+$pf-chart-legend-gutter--Width: 20px;
+$pf-chart-legend-title--FontSize: $pf-chart-global--FontSize;
+$pf-chart-legend-title--Padding: 2px;
+
```

Comment:
In general, each component has their own layout and label props. For example, legend looks like this:

![Screen Shot 2019-05-29 at 11 21 16 AM](https://user-images.githubusercontent.com/17481322/58569319-e4ffe880-8203-11e9-9f36-ed873b97c21c.png)

The layout and label props that are typically applied are:

![Screen Shot 2019-05-29 at 11 22 09 AM](https://user-images.githubusercontent.com/17481322/58569449-1f698580-8204-11e9-8ea3-dccb10439a3c.png)

I'm wondering if these props should be applied globally? Or, should var be defined for each chart component? Thus, allowing a single component to be overriding individually, without effecting other charts.

Reply:
I spoke with @gdoyle1 about this and she said it would be best to keep those global properties global, i.e. font size / font family to ensure consistency across charts.